changelogs/unreleased/gh-9643-origin-in-priv-options.md

```diff
@@ -0,0 +1,6 @@
+## feature/box
```

Comment:
Nit: The commit message and the pull request title has the `config: ` prefix, while the change itself doesn't touch or affect any `config` module code.

Reply:
Changed to `box:`.